agent/stats/queue_test.go

```diff
@@ -189,9 +209,45 @@ func TestQueueAddRemove(t *testing.T) {
 		t.Error("Sum value incorrectly set: ", *memStatsSet.Sum)
 	}
 
+	storageReadStatsSet, err := queue.GetStorageReadStatsSet()
+	if err != nil {
+		t.Error("Error getting storage read stats set:", err)
+	}
+	// assuming min is initialized to math.MaxUint64 then truncated
+	if *storageReadStatsSet.Min == int64(math.MaxInt64) &&
```

Comment:
I'll remove the int64 cast here and in line 217.  These are the result of an overabundance of caution.